fdzipstream.c

```diff
@@ -418,7 +417,7 @@ zs_init ( int fd, ZIPstream *zs )
                              zs_deflate_process,
                              zs_deflate_finish ) )
     {
-      free (_zs);
+      free (zs);
```

Comment:
The intent of my change was to *not* call free() on the client-supplied zs pointer. IMO It's up to the client to free it, depending on how he allocated it. But given how zs_free() works it's implicitly a requirement that all ZIPstream pointers are malloc'ed so I guess it's alright.